Plugins/Flow.Launcher.Plugin.Explorer/Search/ResultManager.cs

```diff
@@ -77,11 +78,12 @@ internal static Result CreateFolderResult(string title, string subtitle, string
                 CopyText = path,
                 Action = c =>
                 {
+                    // open folder
                     if (c.SpecialKeyState.CtrlPressed || (!Settings.PathSearchKeywordEnabled && !Settings.SearchActionKeywordEnabled))
                     {
                         try
                         {
-                            Context.API.OpenDirectory(path);
+                            _openFolder(path);
```

Comment:
is `_openFolder` a function? Let's preserve the naming convention that function should be Pascal Case